app/pods/components/agenda/agendaitem/agendaitem-case/subcase-document/document-link/template.hbs

```diff
@@ -172,40 +172,40 @@
     </div>
   {{/if}}
 </div>
-{{#if this.isUploadingNewVersion}}
```

Comment:
Hier zelfde. Met `isUploadingNewVersion` zeg je niks dat niet waar is imo, en het is duidelijker wat er aan de gang is dan bij `isUploadingNewPiece` ...